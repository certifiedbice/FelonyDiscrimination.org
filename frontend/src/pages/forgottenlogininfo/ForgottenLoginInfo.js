import React from 'react';
import './ForgottenLoginInfo.css';

export default function ForgottenLoginInfo(){
	return(
		<div>
			<h2>Forgotten Login Info Page</h2>
		</div>
	);
}