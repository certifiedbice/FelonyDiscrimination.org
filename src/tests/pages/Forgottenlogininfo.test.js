import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ForgottenLoginInfo from '../../pages/forgottenlogininfo/ForgottenLoginInfo';

it('renders without crashing',()=>{
    const div=document.createElement('div');
    ReactDOM.render(<ForgottenLoginInfo/>,div);
});

it('renders correctly',()=>{
    const tree=renderer
        .create(<ForgottenLoginInfo/>)
        .toJSON();
        expect(tree).toMatchSnapshot();
});