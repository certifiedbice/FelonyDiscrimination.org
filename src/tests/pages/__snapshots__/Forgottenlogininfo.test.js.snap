// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h2>
    Forgotten Login Info Page
  </h2>
</div>
`;
